rule-engines/jess/README.MD

```diff
@@ -0,0 +1,10 @@
+## Jess
```

Comment:
We can do without the README file since there's one in the parent rule-engines module already that will cover this and the other articles. I'll add this one to it after we publish.

Reply:
Will do. I noticed that the existing readme had the sentence: "Articles specific to a given rule engine go in the relevant submodule.", but will remove the readme from Jess as requested.